release-notes/schemas/dotnet-os-packages.json

```diff
@@ -0,0 +1,190 @@
+{
+  "description": "The set of packages required by a given product version for a set of distros.",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "channel-version": {
+      "description": "Major (or major.minor) version of product.",
+      "type": "string"
+    },
+    "eol-date": {
+      "description": "End of life date of product version.",
+      "type": "string",
+      "format": "date"
+    },
+    "packages": {
+      "description": "Set of nominal packages used by product, with descriptions.",
+      "type": "array",
+      "items": {
+        "description": "A nominal package is a distro-agnostic representation of a package, including the scenarios for which the package is required. A nominal package will be referenced a distribution package, with a distribution-specific package name.",
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "id": {
+            "description": "Id of nominal package.",
+            "type": "string"
+          },
+          "name": {
+            "description": "Display name of nominal package.",
+            "type": "string"
+          },
+          "required-scenarios": {
+            "description": "Required scenarios for which the package must be used.",
+            "type": "array",
+            "items": {
+              "description": "Scenarios relating to package dependencies. \u0027All\u0027 includes both CoreCLR and NativeAOT while \u0027Runtime\u0027 is intended to cover CoreCLR, only.",
+              "enum": [
+                "all",
+                "runtime",
+                "https",
+                "cryptography",
+                "globalization",
+                "kerberos"
+              ]
+            }
+          },
+          "references": {
+            "description": "Related references.",
+            "type": [
+              "array",
+              "null"
+            ],
+            "items": {
+              "type": [
+                "string",
+                "null"
+              ]
+            }
+          }
+        },
+        "required": [
+          "id",
+          "name",
+          "required-scenarios",
+          "references"
```

Comment:
`references` probably doesn't need to be required.

Reply:
Agreed. It is `IList<string>?`.

https://github.com/richlander/distroessed/blob/f180c74df71817868df497d58dfd3ecce50c4e1c/src/DotnetRelease/PackageRecords.cs#L32

I'm not sure what to do.

Reply:
Making the corresponding parameter optional should do the trick (i.e. assign to `null`)

Reply:
Thanks!